Cargo.toml

```diff
@@ -11,4 +12,13 @@ edition = "2018"
 
 [dependencies]
 async-trait = "0.1"
-futures = "0.3"
+tokio = { version = "0.2", features = ["fs", "rt-threaded", "macros"] }
```

Comment:
I made `tokio` a default feature now, so it can be disabled with `--no-default-features --features use_futures_types` or `{ ..., default-features = false, features = ["use_futures_types"] }` in Cargo.toml.